dcos-labs/http-server
# DCOS-42974: Handle proxy errors

1. Merged the proxy-secure flag branch
2. Added proxy error handling https://github.com/dcos-labs/http-server/pull/1/commits/5aa9afd91144fd1955247ca19c3cf6060dcdfb1e

## Testing:
1. Checkout
2. Launch the server with a valid URL that doesn't exist `node ./bin/http-server -P http://dcos.ios`
3. http://127.0.0.1:8080
4. Verify that server didn't crash

## Trade-offs
1. Tests are failing and they are failing on the `proxy-secure-flag` branch as well
2. Why not upstream? It's been open in https://github.com/indexzero/http-server for years with no particular outcome.
